src/app/query/stacks/balance/stx-balance.query.ts

```diff
@@ -55,7 +55,7 @@ export function useAnchoredStacksAccountBalanceQuery<T extends unknown = FetchAc
 
   return useQuery({
     enabled: !!address,
-    queryKey: ['get-address-anchored-stx-balance', address],
+    queryKey: ['get-address-anchored-stx-balance', address, client],
```

Comment:
I wonder if its better to use `network.chain.stacks.url` here? instead of client? or its ok as is?

Reply:
The values should be serialisable, so it's better to have a string value in here

Reply:
no, `network.id` will be better